recipes/glib/all/conanfile.py

```diff
@@ -194,6 +194,7 @@ def package_info(self):
             self.cpp_info.components["gio-unix-2.0"].libs = ["gio-2.0"]
             self.cpp_info.components["gio-unix-2.0"].requires.extend(["gobject-2.0", "gio-2.0"])
             self.cpp_info.components["gio-unix-2.0"].includedirs = [os.path.join("include", "gio-unix-2.0")]
+        self.env_info.GLIB_COMPILE_SCHEMAS = os.path.join(self.package_folder, "bin", "glib-compile-schemas")
```

Comment:
Do we need to care about forward/backward slashes?
This variable might be used on winbash.
```suggestion
        self.env_info.GLIB_COMPILE_SCHEMAS = os.path.join(self.package_folder, "bin", "glib-compile-schemas").replace("\\", "/")
```

Reply:
I am not on Windows to test this. But I am pretty sure `os.path.join` is cross-platform.

Reply:
Completely true, but on winbash. bash doesn't interpret backslashes as a path separator.

Reply:
Are you sure Python doesn't return the correct formatting (no backslashes) in that case? Does calling `python -c 'import os; print(os.getcwd())'` in win-bash, Python return a path with backslashes?

Reply:
It will return correct formatting when running conan under winbash,
but will not when running under cmd/powershell and some shell script needs to use it in a winbash.

Reply:
I see. I don't know how likely that issue would be so if you think it can happen then let me know and I will commit the suggestion. Do we need to also handle `C:\` vs `/c/` in that scenario?

Reply:
Applying `tools.unix_path` should not be done in general, because you don't know beforehand whether some dependency requires a winbash path (or not).
My suggestion is just a precaution for a potential prolem.
I don't know of a dependency using this GLIB_COMPILE_SCHEMAS variable (yet).
When/if the problem arises, then we can look at it again.

Reply:
@madebr Ok sounds good. I was just wondering if that would be enough to solve the problem potential problem. If path is `C:\some\path` I am wondering if passing `C:/some/path` to win-bash if it still would crash or not.

Reply:
In my experience, replacing backslashes with forward slashes works.
I still use `tools.unix_path` if the consumer is 100% a shell script.